### PR TITLE
add instructions for waiting after the gateway is reloaded

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**

### CODE REVIEW

File: _docs/tasks/traffic-management/ingress.md [hunk 1]
```diff
@@ -269,6 +269,8 @@ Here we use _curl_'s `-k` option to instruct _curl_ not to check our certificate
    content-length: 0
    x-envoy-upstream-service-time: 6
    ```
+   > Note that it may take time for the new gateway defintion to propagate and you may get the following error:
```
Comment (andraxylia) [inline _docs/tasks/traffic-management/ingress.md]: There is a typo in definition, this is why circleci fails.

**Comment by linsun** [comment]
@vadimeisenbergibm could you resolve merge conflict?

**Comment by vadimeisenbergibm** [comment]
Closing the pull request, too hard to merge it.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 7d4fb85b382dced23e7175123752c94a2399514c

https://app.netlify.com/sites/preliminary-istio/deploys/5b101e53dd6a5443fb7adc45

**Pull request closed without merging**